rust/Cargo.toml.in

```diff
@@ -58,6 +58,8 @@ lru = "~0.12.5"
 der-parser = { version = "~9.0.0", default-features = false }
 kerberos-parser = { version = "~0.8.0", default-features = false }
 
+tree_magic_mini = { version = "~3.1.6", features = [@WITH_GPL_DATA@] }
```

Comment:
So the support is always compiled in? What happens if the data is not there?

Reply:
you can load the mime type def from a file on disk. I'll document that.